src/Cimpress.Nancy.Logging/LoggingBootstrapperExtender.cs

```diff
@@ -89,7 +89,7 @@ public Response OnBeforeRequest(NancyContext context, IDictionary<string, object
             var bodyString = new StreamReader(request.Body).ReadToEnd();
             request.Body.Position = 0;
 
-            var isBodyJson = string.Equals(request.Headers.ContentType, "application/json", StringComparison.OrdinalIgnoreCase);
+            var isBodyJson = string.Equals(request.Headers.ContentType ?? string.Empty, "application/json", StringComparison.OrdinalIgnoreCase);
```

Comment:
I think a more robust fix would be to require the Content-Type header for POST/PUT/PATCH methods and have it be optional for OPTIONS/HEAD/GET